src/main/js/helpers/label.js

```diff
@@ -338,7 +338,8 @@ const createTooltipDiv = () => {
  */
 const destroyTooltipDiv = () => {
     if (document.querySelector(`.${styles.labelPopupTooltip}`)) {
-        document.querySelector(`.${styles.labelPopupTooltip}`).remove();
+        const element = document.querySelector(`.${styles.labelPopupTooltip}`);
+        element.parentNode && element.parentNode.removeChild(element);
```

Comment:
are we wanting to return `true` or possibly `false` from this?  If so, for what reason?

would it be more appropriate to use
```javascript
if(element.parentNode) {
   element.parentNode.removeChild(element);
}
```

Reply:
It won't return any boolean value Andy. It will just remove the element present in DOM. 

**element.parentNode && element.parentNode.removeChild(element);**
It is like kind of if statement if node is present then it will execute the logic else No and it won't make any difference either way i guess Andy.

Reply:
I think if we are attempting to validate that `element.parentNode` exists and then retrieve the child node, I would prefer to do that with an explicit if block

Reply:
Fixed [here](https://github.com/cerner/carbon-graphs/pull/309/files#diff-ee13bfd863a6dec0adb29b9fc41caf73R342).